### DISCUSSION TITLE
apt deb archive?

### DISCUSSION THREAD

**Discussion by macerg** [termux/termux-packages#13356] [Q&A] [answered]
Does anyone know of a termux apt *.deb archive online?

I want to install a back release of an application. 
I inadvertently deleted my cached versions by doing a apt autoclean.

**Comment by sylirre** [comment, marked as answer]
You may ask for a specific package version in https://github.com/termux-user-repository/tur.

**Comment by macerg** [comment]
So how would I ask for firefox/x11 aarch64 105.0.3
 firefox_105.0.3_aarch64.deb
for example?

**Reply by sylirre** [reply]
Open issue, describe that you need package "firefox" with version "105.0.3" and reason why that version is needed.

**Comment by macerg** [comment]
OK, thanks for the information.
